lib/web_ui/lib/src/engine/compositor/fonts.dart

```diff
@@ -14,25 +14,76 @@ const String _robotoUrl =
 
 /// Manages the fonts used in the Skia-based backend.
 class SkiaFontCollection {
-  final List<Future<ByteBuffer>> _loadingFontBuffers = <Future<ByteBuffer>>[];
-  final List<Uint8List> _dynamicallyLoadedFonts = <Uint8List>[];
+  final List<Future<_RegisteredFont>> _unloadedAssetFonts =
```

Comment:
Suggestion: naming not clear, add comment or rename.

Reply:
Done.